telegram-bot/src/main/java/com/exadel/telegrambot/bot/utils/Constant.java

```diff
@@ -0,0 +1,26 @@
+package com.exadel.telegrambot.bot.utils;
+
+
+public class Constant {
+    protected String SKIP = "Skip";
```

Comment:
Lets use public static field, instead of protected and extending

Reply:
yes, it will be better I got you thanks